templates/circuit.yaml

```diff
@@ -64,40 +62,15 @@ spec:
           - name: combined-params
             path: /app/combined_params.json
 
-    # Build circuit for ansatz
-    - name: build-ansatz-circuit
-      parent: generic-task
-      inputs:
-        parameters:
-          - name: command
-            value: python3 main_script.py
-        artifacts:
-          - name: ansatz
-            path: /app/ansatz.json
-          - name: params
-            path: /app/params.json
-          - name: main-script
-            path: /app/main_script.py
-            raw:
-              data: |
-                from zquantum.core.circuit import (build_ansatz_circuit,
-                                                   load_circuit_template,
-                                                   load_circuit_template_params,
-                                                   save_circuit)
-                ansatz = load_circuit_template('ansatz.json');
-                params = load_circuit_template_params('params.json');
-                circuit = build_ansatz_circuit(ansatz, params);
-                save_circuit(circuit, 'circuit.json')
-      outputs:
-        artifacts:
-          - name: circuit
-            path: /app/circuit.json
-
```

Comment:
How come you are deleting this template? I don't see an equivalent replacement and I believe this template is needed for doing final state preparation + measurements

Reply:
Since now Ansatz is an interface, we use `ansatz-specs` as an input to `optimize-varational-circuit` task.
I modified workflows for hydrogen and QAOA and didn't need to use that.

Reply:
We still need it for workflows that do an optimization, then take the optimal parameters and run the final circuit separately so I think this task is necessary for such workflows

Reply:
For example, if someone wanted to run VQE optimization using a simulator, but then estimate the energy using a real QPU

Reply:
Makes sense 👍

Reply:
Added this template in [34bd35](https://github.com/zapatacomputing/z-quantum-core/pull/42/commits/34bd35a8c57c1c7e3608d359ff0a996288983fa5)